mergify_engine/tasks/engine/v2.py

```diff
@@ -141,6 +141,13 @@ def run_actions(installation_id, installation_token, subscription,
 def handle(installation_id, installation_token, subscription,
            pull_request_rules_raw, event_type, data, pull_raw):
 
+    # Some mandatory rules
+    pull_request_rules_raw["rules"].extend([
+        {"name": "delete backport branch (Mergify rule)",
+         "conditions": ["head~=^mergify/bp/", "closed"],
```

Comment:
Maybe add also author?
Not sure we have a config variable available with that author name available though.